x509util/name.go

```diff
@@ -131,24 +149,44 @@ type DistinguishedName struct {
 	Value interface{}      `json:"value"`
 }
 
+// newDistinguisedNames returns a list of DistinguishedName with the attributes not
+// present in attributeTypeNames.
 func newDistinguisedNames(atvs []pkix.AttributeTypeAndValue) []DistinguishedName {
```

Comment:
Maybe the function name can reflect the fact that it excludes the attribute names? It might get overly long doing so though, like `newDistinguishedNamesExcludingExistingAttributes`. Don't have a better proposal atm 😅

Reply:
Fixed with c93d3e3 (`newExtraNames`)